plugins/woocommerce/tests/e2e-pw/tests/smoke-tests/update-woocommerce.spec.js

```diff
@@ -8,17 +8,23 @@ const {
 	deleteZip,
 } = require( '../../utils/plugin-utils' );
 
+const skipMessage = 'Skipping this test because UPDATE_WC is not "true"';
+
 let pluginZipPath;
 
+test.skip( () => {
+	const shouldSkip = UPDATE_WC !== 'true';
+
+	if ( shouldSkip ) {
+		console.log( skipMessage );
+	}
+
+	return shouldSkip;
+}, skipMessage );
+
 test.describe.serial(
 	'WooCommerce plugin can be uploaded and activated',
 	() => {
-		// Skip test if UPDATE_WC is falsy.
-		test.skip(
-			! Boolean( UPDATE_WC ),
-			`Skipping this test because UPDATE_WC is falsy: ${ UPDATE_WC }`
-		);
```

Comment:
I replaced this ☝️  with [this](https://github.com/woocommerce/woocommerce/pull/35760/files#r1042061263) in order to show the skip message on the Allure reports too.